test/extended/storage/storage_networkpolicy.go

```diff
@@ -296,6 +363,55 @@ var _ = g.Describe("[sig-storage][OCPFeature:StorageNetworkPolicy] Storage Netwo
 		runResourceChecks(oc, CSIResourcesToCheck, currentPlatform)
 	})
 
+	g.It("should verify required labels for LSO related resources", func() {
+		// Skip if LSO is not installed or version is lower than 4.21.0
+		if !lsoInstallInfo.Installed {
+			g.Skip("LSO is not installed on this cluster")
+		} else if !isLSOVersionSupported(lsoInstallInfo.Version) {
+			g.Skip(fmt.Sprintf("LSO network policy support requires version >= 4.21.0, current version: %s", lsoInstallInfo.Version))
+		}
+
+		LSOResourcesToCheck := []resourceCheck{
```

Comment:
Maybe we could also consider append it to `CSIResourcesToCheck` if lso is installed.

Reply:
Thanks for the comment. I once considered combining, but it will give us the clear result from the case level (skipped if LSO is not installed), so I think we can keep this :)